package.json

```diff
@@ -81,7 +81,7 @@
     "karma-script-launcher": "^0.1.0",
     "loopback-boot": "^2.7.0",
     "loopback-datasource-juggler": "^2.19.1",
-    "loopback-testing": "^1.1.0",
+    "loopback-testing": "~1.1.0",
```

Comment:
Is there any particular reason for locking down loopback-testing version to 1.1.x?